Validation/HGCalValidation/interface/CaloParticleSelector.h

```diff
@@ -0,0 +1,139 @@
+#ifndef Validation_HGCalValidation_CaloParticleSelector_h
+#define Validation_HGCalValidation_CaloParticleSelector_h
+/* \class CaloParticleSelector
+ *
+ * \author Giuseppe Cerati, INFN
+ *
+ *  $Date: 2013/05/14 15:46:46 $
```

Comment:
is this copied from somewhere?

Reply:
Yes, the work is based on the MultiTrackValidator where we took the design and this is from the TrackingParticleSelector.h which for us is CaloParticleSelector. We mentioned that in the PR description, really no intention to avoid giving credit. I will continue to the rest of the comments.

Reply:
Okay, I was just wondering about the date on a new file being 2013.